app/services/projects_overview_service.rb

```diff
@@ -118,15 +135,19 @@ def sortable_columns
     }
   end
 
-  def sort(records, params)
-    order = params[:order]&.values&.first
+  def sort(records)
+    order_state = @view_state.state['table']['order'][0]
+    order = @params[:order]&.values&.first
     if order
       dir = order[:dir] == 'desc' ? 'DESC' : 'ASC'
       column_index = order[:column]
     else
       dir = 'ASC'
       column_index = '1'
     end
+    if order_state != [column_index.to_i, dir.downcase]
+      @view_state.state['table']['order'][0] = [column_index.to_i, dir.downcase]
```

Comment:
Is this line totally correct? In default state, the order is like this: `order: [2, 'asc']`, but here, it changes to `order: [[2, 'asc'], 'asc']`.

Reply:
Yes, this is correct, order should contain two-dimensional array, I will fix default value